homeassistant/components/squeezebox/browse_media.py

```diff
@@ -78,83 +94,209 @@
     "Playlists": MediaType.PLAYLIST,
     "Genres": MediaType.GENRE,
     "Favorites": None,  # can only be determined after inspecting the item
+    "Apps": MediaClass.APP,
+    "Radios": MediaClass.APP,
+    "App": None,  # can only be determined after inspecting the item
     "New Music": MediaType.ALBUM,
+    MediaType.APPS: MediaType.APP,
+    MediaType.APP: MediaType.TRACK,
 }
 
 
+@dataclass
+class Squeezebox_maps:
+    """Class for browser to squeezebox mappings."""
+
+    content_type_to_child_type: dict[
+        str | MediaType,
+        str | MediaType | None,
+    ]
+    content_type_media_class: dict[str | MediaType, dict[str, MediaClass | None]]
+    squeezebox_id_by_type: dict[str | MediaType, str]
+    media_type_to_squeezebox: dict[str | MediaType, str]
+
+
 async def build_item_response(
     entity: MediaPlayerEntity,
     player: Player,
     payload: dict[str, str | None],
     browse_limit: int,
+    known_apps: set[str],
+    squeezebox_maps: Squeezebox_maps,
 ) -> BrowseMedia:
     """Create response payload for search described by payload."""
 
+    def update_squeezebox_maps(cmd: str | MediaType, type: str) -> None:
+        """Add items to maps for new apps or radios."""
+        squeezebox_maps.media_type_to_squeezebox.update({cmd: cmd})
+        squeezebox_maps.squeezebox_id_by_type.update({cmd: type})
+        squeezebox_maps.content_type_media_class.update(
+            {
+                cmd: {
+                    "item": MediaClass.DIRECTORY,
+                    "children": MediaClass.TRACK,
+                }
+            }
+        )
+        squeezebox_maps.content_type_to_child_type.update({cmd: MediaType.TRACK})
+
+    def build_response_apps_radios_category(
+        _cmd: str | MediaType,
```

Comment:
Similarly, adding an underscore to a parameter is typically used to indicate that it's use is non-public / part of an internal usage not meant to be exposed. That's not the case here, so it should be removed.